presto-jdbc/src/main/java/com/facebook/presto/jdbc/ColumnInfo.java

```diff
@@ -227,6 +233,8 @@ private static int getType(TypeSignature type)
                 return Types.DATE;
             case "decimal":
                 return Types.DECIMAL;
+            case "float":
+                return Types.REAL;
```

Comment:
in the JDBC spec, Types.FLOAT is equivalent to the Java double type. So we need REAL here. see: http://docs.oracle.com/javase/1.5.0/docs/guide/jdbc/getstart/mapping.html